datahub-web-react/src/app/entityV2/shared/tabs/Documentation/DocumentationTab.tsx

```diff
@@ -13,10 +13,9 @@ import { REDESIGN_COLORS } from '@app/entityV2/shared/constants';
 import { DescriptionEditor } from '@app/entityV2/shared/tabs/Documentation/components/DescriptionEditor';
 import { DescriptionPreviewModal } from '@app/entityV2/shared/tabs/Documentation/components/DescriptionPreviewModal';
 import { LinkList } from '@app/entityV2/shared/tabs/Documentation/components/LinkList';
-import { Editor } from '@app/entityV2/shared/tabs/Documentation/components/editor/Editor';
 import { getAssetDescriptionDetails } from '@app/entityV2/shared/tabs/Documentation/utils';
 import { EDITED_DESCRIPTIONS_CACHE_NAME } from '@app/entityV2/shared/utils';
-import { Button } from '@src/alchemy-components';
+import { Button, Editor } from '@src/alchemy-components';
```

Comment:
can we make it so that the editor header is stickie, and so that it expands across the width of the content?

this should work and act like it does in the documentation module on the home page (scroll with toolbar staying at the top, border under the toolbar spans the width of things)